.chronus/changes/remove_deprecated_description_details-2024-8-18-12-0-24.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: deprecation
```

Comment:
should this be marked deprecation or just bug fix since we deprecated it in a previous version